doc/userguide/rules/xbits.rst

```diff
@@ -21,9 +20,9 @@ Notes
    ``track ip_dst``, if you want to match on the server response,
    you check it (``isset``) with ``track ip_src``.
 
--  To not alert, use ``noalert;``
+-  To not alert, use ``noalert`` as a standalone option to ``xbits`` just like flowbits.
 
-- the ``toggle`` option will flip the value of the xbits.
+-  the ``toggle`` option will flip the value of the xbits.
```

Comment:
I think this should be `will flip the value of the xbit` (singular, not plural). But this also looks like a white space change that may be hear by accident?

Reply:
yep. Thank you!